test/e2e_node/container_lifecycle_test.go

```diff
@@ -1234,7 +1223,7 @@ var _ = SIGDescribe(framework.WithSerial(), "Containers Lifecycle", func() {
 				}
 
 				ginkgo.By("restarting the kubelet")
-				startKubelet()
+				restartKubelet()
```

Comment:
why restart? The kubelet is alreasy stopped on line 1202

Reply:
Because `stopKubelet` [returns restart function](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/util.go#L457-L461).